sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo/Azure.Monitor.OpenTelemetry.LiveMetrics.Demo.csproj

```diff
@@ -12,7 +12,11 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\Azure.Monitor.OpenTelemetry.LiveMetrics\src\Azure.Monitor.OpenTelemetry.LiveMetrics.csproj" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" />
```

Comment:
Curious why we need this reference. `IOpenTelemetryBuilder` is part of SDK so in order to author\target a "Use" extension you should only need an SDK reference. Was there something else we needed?

Reply:
`OpenTelemetry.Extensions.Hosting` package has reference to [Microsoft.Extensions.Hosting.Abstractions](https://www.nuget.org/packages/Microsoft.Extensions.Hosting.Abstractions/), but `Host.CreateApplicationBuilder` is available in `Microsoft.Extensions.Hosting` package.

This is a demo console app.